src/utils/src/bytes.rs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: I don't know if bytes is the best name for this module. Maybe `byte_order` or `endianness` or something else.